image/image.go

```diff
@@ -11,6 +11,7 @@ import (
 
 var validHex = regexp.MustCompile(`^([a-f0-9]{64})$`)
 
+// Image stores the image configuration
```

Comment:
Again, please document the usage of the type. The goal is not to have `golint` pass.